src/Auth/Token.php

```diff
@@ -105,6 +124,24 @@ public function refreshToken()
     }
 
     /**
+     * Retrieve data from configuration
+     */
+    protected function retrieveConfig()
+    {
+
+        try {
+            self::$accessToken = $this->config->accessToken();
+            self::$refreshToken = $this->config->refreshToken();
+            self::$expiresAt = Carbon::createFromTimestamp($this->config->expiresAt());
+        } catch(AuthNotDefinedException | AuthInternalNotDefinedException $e) {
```

Comment:
Só o Null Coalesce não consegue resolver nesse caso? Tipo `$this->config->accessToken() ?? self::$accessToken ?? null;` ?

Reply:
Acho que não; essas funções lançam exceção.